src/spacepackets/ccsds/tm_frame.py

```diff
@@ -21,8 +21,7 @@ class TransferFrameDataFieldStatus:
     segment_len_id: int
     first_header_pointer: int
 
-    @classmethod
-    def pack(cls, data: TransferFrameDataFieldStatus) -> bytes:
+    def pack(self, data: TransferFrameDataFieldStatus) -> bytes:
```

Comment:
almost there :) the data argument is not required anymore, and data can be replaced with self